### ISSUE TITLE
update R/mergeData.R

### ISSUE THREAD

**Issue by lwaldron** [waldronlab/curatedMetagenomicData#195]
https://github.com/waldronlab/curatedMetagenomicData/blob/legacy/R/mergeData.R

**Comment by sdgamboa** [comment]
Function updated in pull request #202

**Comment by schifferl** [comment]
This has been implemented in 09b1b89, please see #202 for more details. The documentation will move along, but try these examples in the mean time:

```r
curatedMetagenomicData("AsnicarF_20.+.pathway_abundance", print = FALSE) |>
    sapply(curatedMetagenomicData, dryrun = FALSE) |>
    mergeData()
```

```r
curatedMetagenomicData("AsnicarF_20.+.relative_abundance", print = FALSE) |>
    sapply(curatedMetagenomicData, dryrun = FALSE) |>
    mergeData()
```

Closing this issue with the new implementation.
